sources/Debate-frontend/src/app/loginpage/registration/registration.component.ts

```diff
@@ -12,27 +14,32 @@ import { NgForm } from '@angular/forms';
 export class RegistrationComponent implements OnInit {
   model: UserDto;
   passwordCheck: string;
+  account: UserDto;
 
-  @ViewChild('regform') regform: NgForm;
+  @ViewChild("regform") regform: NgForm;
 
-  constructor(private userService: UserService, private router: Router) { }
+  constructor(private userService: UserService, private router: Router, private authService: AuthService) { }
 
   ngOnInit(): void {
-      this.model = { login: '', password: '', email: '', role: 'USER' };
+    this.model = {login: '', password: '', email: '', role: 'USER'};
+    this.account = {login: '', password: '', email: '', role: ''};
   }
 
   onSubmit(){
-    if (!this.regform.form.valid){
+    if (!this.regform.form.valid) {
       return;
     }
-    if (this.regform.controls.password.value === this.regform.controls.passwordCheck.value){
+    if (this.regform.controls.password.value === this.regform.controls.passwordCheck.value) {
       this.model.login = this.regform.controls.login.value;
       this.model.password = this.regform.controls.password.value;
       this.model.email = this.regform.controls.email.value;
-      // todo when successful registration then log-in user to new account
-      this.userService.addUser(this.model).subscribe(
-        res => this.router.navigate(['/home']), 
-        err => console.log(err)
+      this.account.login = this.model.login;
+      this.account.password = this.model.password;
+      this.userService.addUser(this.model).subscribe( 
+        tmp => {this.authService.logToAccount(this.account), 
```

Comment:
```suggestion
        tmp => {
                       this.authService.logToAccount(this.account);
```

Semicolon instead of comma